src/static/styles/partials/components/_wf-tools-devtools.scss

```diff
@@ -1,52 +1,134 @@
-.wf-tools-panel {
```

Comment:
The way the majority of styles are working is that a component should be a specific element and the filename should match up to the classname.

This looks like it is styles specific to the page. Please just make a top level stylesheet and add these there.